doc/cmd_run.rst

```diff
@@ -271,6 +272,42 @@ from a task while it is being executed,
 this is controlled by the ``verbosity`` param.
 
 
+check_file_uptodate
+-------------------
+
+`doit` provides different options to check if files are up to date.
+Use the option ``--check_file_uptodate`` to choose:
+
+ * md5: use the md5sum of the file.
+ * timestamp: use only the timestamp and size.
```

Comment:
use only timestamp, no size. I guess most people would expect something like on `make` where target timestamp is compared. That's not the case, so please mention exactly how it works. i.e. any timestamp change counts as a change in the file_dep.